util/tar.go

```diff
@@ -86,14 +91,21 @@ func Unpack(tarFile string) ([]string, string, error) {
 
 			// if it's a newFile create it
 		case tar.TypeReg:
+			// #nosec G304 -- File path is coming from inside the TAR file.
 			newFile, err := os.OpenFile(target, os.O_CREATE|os.O_RDWR, os.FileMode(header.Mode))
```

Comment:
Actually, this could be a potential vulnerability, which we should investigate. If the tar file contains something like `'../../../../../../../../../../etc'`, which points outside the extraction directory.

See this [similar case](https://github.com/advisories/GHSA-47wr-426j-fr82)